src/pages/Main.jsx

```diff
@@ -1,7 +1,79 @@
+import { Stack } from "@mui/system";
+import { styled } from "@mui/material";
+import Typography from "@mui/material/Typography";
 import Layout from "../components/Layout/Layout";
+import CarouselSilder from "../components/component/CarouselSilder";
 
 const Main = () => {
-  return <Layout>메인페이지</Layout>;
+  return (
+    <Layout>
+      <Stack>
+        <Stack>
+          <CarouselSilder />
+        </Stack>
+        <ContentWrapper>
+          <TitleOne>교육 표어</TitleOne>
```

Comment:
```suggestion
          <TitleOne>교육 표어</TitleOne>
```
일단 네이밍부터 지양해야할 부분임. Titile1 Title2 이런 네이밍은 절대 지양해야함
MainTitle, SubTitle과 같이 네이밍부터 정해야하고, 재사용가능한 컴포넌트로 만들어서 components 경로에 common 또는 shared 폴더 만들어서 다른 페이지에서도 쓸수 있도록 하는게 베스트임.